### PR TITLE
devnet release and fix fullnodes

### PR DESCRIPTION
**Pull request by sherry-x**
### Description

- fix symlink for fullnodes
- add missing indexer variables

### Test Plan

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1828)

### CODE REVIEW

File: terraform/fullnode/aws/variables.tf [hunk 1]
```diff
@@ -120,4 +120,20 @@ variable "utility_instance_type" {
 variable "fullnode_instance_type" {
   description = "Instance type used for validator and fullnodes"
   default     = "c5.xlarge"
+}
+
+variable "enable_indexer" {
```
Comment (rustielin) [inline terraform/fullnode/aws/variables.tf:125]: Didn't realize we gave the option to run indexer on fullnode cluster. Looks like it snuck in here: https://github.com/aptos-labs/aptos-core/commit/ad6ee9573cb4941fc4bdeb8bba1189d3d1c7d27b#diff-2c02aa155c3124ea6525b43e846d6479f8eeb36ba10b5ca00b6d03e141ce67a4 when we reverted an old change 🤷 as long as it works ig

**Review by rustielin — APPROVED** [review]
LGTM! Thanks for the fix!

File: terraform/fullnode/aws/variables.tf (hunk 1 above)
Comment (sherry-x) [reply terraform/fullnode/aws/variables.tf:125]: I can remove it if not needed anymore

Comment (rustielin) [reply terraform/fullnode/aws/variables.tf:125]: yea can you also remove `indexer.tf` ? it's not being used anywhere

**Comment by gregnazario** [comment]
@sherry-x can we close this?

**Comment by davidiw** [comment]
This actually seems to be somewhat relevant, as there are a few broken symlinks. Maybe we can help out by sending an updated PR :P

**Pull request merged**